<!-- Sleiman Safaoui
     July 13, 2017
   -->

<launch>

  <arg name="img_src" default="w"/>

  <node pkg="balls_detector" type="modify_camera_param.py" name="modify_camera_parameters"></node>

  <node pkg="balls_detector" type="cam_bridge.py" name="cam_bridge">
    <param name="img_src" value="$(arg img_src)"/>
  </node>

  <node pkg="balls_detector" type="tagger" name="tagger">
    <rosparam command="load" file="$(find balls_detector)/launch/tag_param.yaml"/>
  </node>

  <node pkg="balls_detector" type="balls_detector_node" name="balls_detector_node">
    <rosparam command="load" file="$(find balls_detector)/launch/detect_param.yaml"/>
  </node>

  <node pkg="balls_detector" type="tracker.py" name="tracker">
    <rosparam command="load" file="$(find balls_detector)/launch/track_param.yaml"/>
  </node>

</launch>
